example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -159,7 +171,7 @@ MM_CollectorLanguageInterfaceImpl::markingScheme_scanObject(MM_EnvironmentBase *
 			_markingScheme->markObject(env, slot);
 		}
 	}
-	return objectSize;
+	return objectModel->getSizeInBytesWithHeader(objectPtr);
```

Comment:
Why not just call env->getExtensions()->objectModel.getSizeInBytesWithHeader(objectPtr); here?